Vertical.HubSpot.Api/Engagements/HubspotEngagement.cs

```diff
@@ -0,0 +1,171 @@
+        [JsonProperty("id")]
+        public long Id { get; set; }
+
+        [JsonProperty("portalId")]
+        public long PortalId { get; set; }
+
+        [JsonProperty("active")]
+        public bool Active { get; set; }
+
+        [JsonProperty("createdAt")]
+        public long CreatedAt { get; set; }
+
+        [JsonProperty("lastUpdated")]
+        public long LastUpdated { get; set; }
+
+        [JsonProperty("createdBy")]
+        public long CreatedBy { get; set; }
+
+        [JsonProperty("modifiedBy")]
+        public long ModifiedBy { get; set; }
+
+        [JsonProperty("ownerId")]
+        public long OwnerId { get; set; }
+
+        [JsonProperty("type")]
+        public string Type { get; set; }
+
+        [JsonProperty("uid")]
+        public string Uid { get; set; }
+
+        [JsonProperty("timestamp")]
+        public long Timestamp { get; set; }
+
+        [JsonProperty("teamId")]
+        public long TeamId { get; set; }
+
+        [JsonProperty("allAccessibleTeamIds")]
+        public List<long> AllAccessibleTeamIds { get; set; }
+
+        [JsonProperty("bodyPreview")]
+        public string BodyPreview { get; set; }
+
+        [JsonProperty("queueMembershipIds")]
+        public List<object> QueueMembershipIds { get; set; }
+
+        [JsonProperty("bodyPreviewIsTruncated")]
+        public bool BodyPreviewIsTruncated { get; set; }
+
+        [JsonProperty("bodyPreviewHtml")]
+        public string BodyPreviewHtml { get; set; }
+
+        [JsonProperty("gdprDeleted")]
+        public bool GdprDeleted { get; set; }
+    }
+
+    public class HubspotEngagementMetadata
+    {
+        [JsonProperty("startTime")]
+        public long StartTime { get; set; }
+
+        [JsonProperty("endTime")]
+        public long EndTime { get; set; }
+
+        [JsonProperty("title")]
+        public string Title { get; set; }
+
+        [JsonProperty("body")]
+        public string Body { get; set; }
+
+        [JsonProperty("externalUrl")]
+        public Uri ExternalUrl { get; set; }
+
+        [JsonProperty("source")]
+        public string Source { get; set; }
+
+        [JsonProperty("sourceId")]
+        public string SourceId { get; set; }
+
+        [JsonProperty("createdFromLinkId")]
+        public long CreatedFromLinkId { get; set; }
+
+        [JsonProperty("preMeetingProspectReminders")]
+        public List<object> PreMeetingProspectReminders { get; set; }
+    }
+
+    public class HubspotScheduledTask
+    {
+        [JsonProperty("engagementId")]
+        public long EngagementId { get; set; }
+
+        [JsonProperty("portalId")]
+        public long PortalId { get; set; }
+
+        [JsonProperty("engagementType")]
+        public string EngagementType { get; set; }
+
+        [JsonProperty("taskType")]
+        public string TaskType { get; set; }
+
+        [JsonProperty("timestamp")]
+        public long Timestamp { get; set; }
+
+        [JsonProperty("uuid")]
+        public string Uuid { get; set; }
+    }
+}
```

Comment:
multiple classes in a single file? ... i would rather see them separated

Reply:
They are separated now.